.github/workflows/build-test-publish-wheel.yml

```
@@ -0,0 +1,48 @@
+# Copyright (c) 2020-2021, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Build, test, and publish a PyPi wheel (to testpypi)
+
+on:
+  pull_request:
```

```suggestion
  push:
```
Will change this back to `push` if the tests succeed.

---

Our dev cycle typically consists of making a `0.n.0` branch around code freeze, and publishing the final release wheel later. Does it make sense to modify the process slightly to create `rc` branches instead?

I wonder if updating the action to trigger on something like is another alternative that works here:
```
on:
  release:
    types: [created]
```